Sources/ObjCSupport/Builders/LOKSizeLayoutBuilder.m

```diff
@@ -10,126 +10,150 @@
 
 #import <LayoutKitObjC/LayoutKitObjC-Swift.h>
 
+@interface LOKSizeLayoutBuilder ()
+
+@property (nonatomic, nullable) LOKAlignment *privateAlignment;
+@property (nonatomic, nullable) LOKFlexibility *privateFlexibility;
+@property (nonatomic, nullable) NSString *privateViewReuseId;
+@property (nonatomic, nullable) Class privateViewClass;
+
+@property (nonatomic, nullable) id<LOKLayout> privateSublayout;
+@property (nonatomic) CGFloat privateMinWidth;
+@property (nonatomic) CGFloat privateMaxWidth;
+@property (nonatomic) CGFloat privateMinHeight;
+@property (nonatomic) CGFloat privateMaxHeight;
+@property (nonatomic) CGFloat privateWidth;
+@property (nonatomic) CGFloat privateHeight;
```

Comment:
`privateWidth` and `privateHeight` aren't used to construct the layout, so we don't need private properties for them

Reply:
It's for the setter/getter and ability to just call it like a property. So there is no storage needed. Do you want me to remove it and call the actual methods? I think I should leave it as-is.